### ISSUE TITLE
E2E : Catalog App Movie Form E2E Test

### ISSUE THREAD

**Issue by xmano** [blockframes/blockframes#3874]
Refactor the Catalog App Movie Form E2E Tests.

This is currently gated by UI component changes. 

Manually running the test for adding a new movie results in blank screen. So this issue will be deferred until the code changes are in place for New Movie Form for Catalog App

**Comment by phpgeekfr** [comment]
Closing this one since catalog app will be reworked
